engine/language-server/src/main/scala/org/enso/languageserver/search/SearchApi.scala

```diff
@@ -76,7 +76,8 @@ object SearchApi {
       position: Position,
       selfType: Option[String],
       returnType: Option[String],
-      tags: Option[Seq[SuggestionKind]]
+      tags: Option[Seq[SuggestionKind]],
+      isStatic: Option[Boolean]
```

Comment:
Can one ask for both - static/instance - methods at once? Or one needs two queries?

Or is it never needed to ask for both types at once?

Reply:
> Can one ask for both - static/instance - methods at once?

If you don't care about the `static` attribute, you just pass `None`. This way the method will return both static and non-static suggestions.